site/public/js/pdf/PDFAnnotateEmbedded.js

```diff
@@ -115,7 +115,8 @@ function renderPageForDownload(pdf, doc, num, targetNum, file_name) {
             doc.addPage();
         }
         pdf.getPage(num).then((page) => {
-            const viewport = page.getViewport({scale:1.5});
+            const scale = 3; // Increase the scale factor for higher resolution
+            const viewport = page.getViewport({scale: scale});
```

Comment:
If all you did was change 1.5 to 3 here we don't need to introduce a new variable. You can do it on one line.